.changeset/pink-ligers-share.md

```diff
@@ -0,0 +1,49 @@
+---
+"astro": minor
+---
+
+Adds experimental rewriting in Astro, via `rewrite()` function and `next()` function.
```

Comment:
```suggestion
Adds experimental rewriting in Astro with a new `rewrite()` function and the middleware `next()` function.
```
Realized "via" is twice in a row. So, might also be nice to highlight that the function is new!